docs/source/api/cterasdk.core.files.rst

```diff
@@ -23,3 +23,4 @@ Submodules
    cterasdk.core.files.recover
    cterasdk.core.files.rename
    cterasdk.core.files.rm
+   cterasdk.core.files.collaboration
```

Comment:
We're trying to maintain an alphabetical order so, please move this line to its correct place

Reply:
I didn't notice that! Fixed.